src/features/views/components/ViewDataTable/columnTypes/LocalSmartSearchColumnType.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box } from '@mui/material';
+import { Check } from '@mui/icons-material';
+import { FC } from 'react';
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import { IColumnType } from '.';
+import theme from '../../../../../theme';
+import { ZetkinViewColumn } from '../../types';
+
+type LocalSmartSearchViewCell = boolean;
+
+export default class LocalSmartSearchColumnType
+  implements IColumnType<ZetkinViewColumn, LocalSmartSearchViewCell>
+{
+  cellToString(cell: LocalSmartSearchViewCell | null): string {
+    return cell ? cell.toString() : '';
+  }
+
+  getColDef(): Omit<GridColDef, 'field'> {
+    return {
+      renderCell: (params) => {
+        return <Cell cell={params.value} />;
+      },
+      type: 'boolean',
+      width: 150,
+    };
+  }
+}
+
+const Cell: FC<{ cell: LocalSmartSearchViewCell }> = ({ cell }) => {
+  if (!cell) {
+    return null;
+  }
+
+  return (
+    <Box
+      alignItems="center"
+      bgcolor={theme.palette.success.light}
```

Comment:
Gorgeous!

Reply:
Great! What is your thoughts on my question above? 

> I noticed just now that there is two column types: LOCAL_SMART_SEARCH and PERSON_QUERY. should they both use this column? or have misunderstood and given the wrong names to things?

Reply:
Oh I'm sorry I missed that question. Lazy reader!

I think they should both use this same column type. I would also prefer that `LOCAL_SMART_SEARCH` is renamed to `LOCAL_QUERY`, but I overlooked it when the column was created and didn't think I'd point it out now.

But if you do go in and make changes to reuse this column type, it would make sense to rename the enum options for consistency as well.

Reply:
Cool! Will refactor :)